osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/SpinnerTicks.cs

```diff
@@ -44,8 +30,8 @@ private void layout()
                     EdgeEffect = new EdgeEffect
                     {
                         Type = EdgeEffectType.Glow,
-                        Radius = 20,
-                        Colour = glowColour,
+                        Radius = 10,
+                        Colour = Color4.Gray.Opacity(0.2f),

```

Comment:
either way it's just specifying a specific colour. i'm not sure of the difference